### PR TITLE
[ENG-3314] feat: support repeat minter

### PR DESCRIPTION
**Pull request by teebszet**
# 🔘 PR Type
- [x] Enhancement

# 📜 Background
depends on: https://github.com/secretkeylabs/sats-connect/pull/50

issue: https://linear.app/xverseapp/issue/ENG-3314

# 🔄 Changes
feat: support new sats-connect version

Impact:
- sats-connect new function
- create inscription screen UI changes

# 🖼 Screenshot / 📹 Video
Include screenshots or a video demonstrating the changes. This is especially helpful for UI changes.

# ✅ Review checklist
Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "@ledgerhq/hw-transport-webusb": "^6.27.13",
     "@phosphor-icons/react": "^2.0.10",
     "@react-spring/web": "^9.6.1",
-    "@secretkeylabs/xverse-core": "3.0.0",
+    "@secretkeylabs/xverse-core": "3.1.1-7e91800",
```
Comment (jordankzf) [inline package.json:13]: How does the suffix work? Will we remove it later on for prod?

File: src/app/screens/createInscription/ContentLabel/index.tsx [hunk 2]
```diff
@@ -151,7 +154,8 @@ function ContentIcon({ type, content, contentType: inputContentType }: Props) {
           {t('BRC20.TITLE')} {t(`BRC20.${brc20Details.op.toUpperCase()}`)}
         </div>
         <Suffix>
-          {brc20Details.tick} - {brc20Details.value}
+          {formatNumber(new BigNumber(brc20Details.value).multipliedBy(repeat).toString())}{' '}
```
Comment (jordankzf) [inline src/app/screens/createInscription/ContentLabel/index.tsx:157]: (nit)

Can we break this down?

File: src/app/screens/createInscription/index.tsx [hunk 3]
```diff
@@ -28,11 +28,100 @@ import type { UTXO } from '@secretkeylabs/xverse-core/types';
...
+  subLabel?: string;
+  value?: number | string | null;
+  fiatCurrency: string;
+  fiatRate: string;
+  repeat?: number;
+}) {
+  if (!value) {
+    return null;
+  }
+  const fiatValue = new BigNumber(value || 0).dividedBy(100e6).multipliedBy(fiatRate).toFixed(2);
```
Comment (jordankzf) [inline src/app/screens/createInscription/index.tsx:95]: (nit)

Can we add a comment here explaining the magic number `100e6` (sats to BRC I'm assuming)? Or better yet, make it a helper function?

**Review by jordankzf — COMMENTED** [review]
I'm not familiar enough with this repo and my nits might slow things down, feel free to ignore them.

I hope I can contribute with manual testing instead.

File: package.json (hunk 1 above)
Comment (teebszet) [reply package.json:13]: yeah, once we can finalise and publish the next xverse-core version.

the suffixed versions are created from CI on each xverse-core PR

**Review by DuskaT021 — APPROVED** [review]

**Pull request merged**